### ISSUE TITLE
Scroll down to open notification panel

### ISSUE THREAD

**Issue by phanirithvij** [markusfisch/PieLauncher#38]
Would like a custom icon which has the notification icon on it and when selected will open the system notifications. This will completely eliminate the need for two finger and also moving up to the notification area to drag it down.

**Comment by markusfisch** [comment]
Interesting idea 🤔 I would very much like to add this!

**Comment by qube-03** [comment]
i vote for this too, since im very used to it because of olauncher

**Comment by ekasprzak** [comment]
Yup, would be great to have a quick way to open notifications (using one hand).

Having a specific icon is one of the ideas, but the downside would be to have one less favorite icon available.

Other idea might be to expand "ignore touches" option, eg. set percentage of the screen that would be ignored by pie menu and that area could be then used to pull down notifications.

**Comment by phanirithvij** [comment]
I agree with you on the one less fav spot. But where can the space be ideally and is there an ignore touches feature in pie launcher already?

**Comment by ekasprzak** [comment]
>is there an ignore touches feature in pie launcher already?

Yup, you have "Ignore touches" option in settings.
4 possibilities there: none, top, bottom, top+bottom.

>where can the space be ideally

In this case I think only bottom would make a practical sense (so let's say ~10-20% of screen bottom would not launch pie, but if you swipe it down it shows notifications.

**Comment by Alias36** [comment]
I created a minimal Android app that provides the notification pull-down functionality requested here: **QuickPanel**

It's a standalone accessibility service that instantly expands the notification panel when launched. To use it with Pie Launcher:
1. Install QuickPanel and enable it in Accessibility settings
2. Add QuickPanel to your Pie Launcher menu
3. Place it at the **bottom position** of your pie menu
4. Now you can "swipe down" to that bottom position to open notifications with one hand

It really does work beautifully, if I say so myself 😁

Features:
- Zero network permissions, zero tracking, zero data collection
- Immediately exits after triggering (no background process)
- GPL v3 open source

GitHub: https://github.com/Alias36/QuickPanel

APK available in the [Releases](https://github.com/Alias36/QuickPanel/releases/tag/v1.0) section

This gives you the swipe-down notification access without needing to reach the top of the screen.

Hope this helps!
